### PR TITLE
feat(preset): add googleapis packages and group presets

### PR DESCRIPTION
**Pull request by ikesyo**
https://www.npmjs.com/search?q=%40google-cloud

### CODE REVIEW

**Comment by rarkins** [comment]
Is this a convenience grouping or recommended that people always group them together because it's likely to get PRs that won't work if you upgrade them separately? We can add the group either way but we needed to decide whether we make it a *default* grouping versus opt-in.

Cc @justinbeckwith for insight

**Comment by JustinBeckwith** [comment]
👋  for the better part, you can run these modules all on different versions, and everything is going to be fine.  They work independently, and we shouldn't have any diamond dependency type problems.  That having been said, we do have a tendency to do semver major releases in batches, typically centered around the yearly deprecation of a major node.js release.   In terms of release patterns, we tend to release *a lot* of patch and minor versions, as we're constantly chasing security patch releases in downstream dependencies. Let me know if there's anything else I can help explain!  (@bcoe who knows more than me)

**Comment by ikesyo** [comment]
Thanks @rarkins and @JustinBeckwith. I'm okay to make it opt-in.

File: lib/config/presets/internal/group.ts [hunk 1]
```diff
@@ -465,6 +465,16 @@ const staticGroups = {
       },
     ],
   },
+  googleapis: {
+    description: 'Group googleapis packages together',
+    packageRules: [
+      {
+        datasources: ['npm'],
```
Comment (rarkins) [inline lib/config/presets/internal/group.ts:472]: Should this be part of the `packages` definition instead?

**Pull request merged**